sources/packages/forms/src/form-definitions/studentadditionaltransportationappeal.json

```diff
@@ -0,0 +1,1124 @@
+                      "modalEdit": false,
+                      "dataGridLabel": false,
+                      "labelPosition": "top",
+                      "description": "",
+                      "errorLabel": "",
+                      "tooltip": "",
+                      "hideLabel": false,
+                      "tabindex": "",
+                      "disabled": false,
+                      "autofocus": false,
+                      "dbIndex": false,
+                      "customDefaultValue": "",
+                      "calculateValue": "",
+                      "calculateServer": false,
+                      "widget": null,
+                      "attributes": {},
+                      "validateOn": "change",
+                      "validate": {
+                        "required": false,
+                        "custom": "",
+                        "customPrivate": false,
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                      },
+                      "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": ""
+                      },
+                      "overlay": {
+                        "style": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                      },
+                      "allowCalculateOverride": false,
+                      "encrypted": false,
+                      "showCharCount": false,
+                      "showWordCount": false,
+                      "properties": {},
+                      "allowMultipleMasks": false,
+                      "addons": [],
+                      "tag": "p",
+                      "id": "eo0wfja",
+                      "className": "",
+                      "tags": []
+                    },
+                    {
+                      "label": "Upload supporting documents (optional):",
+                      "customClass": "font-weight-bold",
+                      "hideLabel": true,
+                      "tableView": false,
+                      "storage": "url",
+                      "dir": "Addition Transportation",
+                      "webcam": false,
+                      "fileTypes": [
+                        {
+                          "label": "",
+                          "value": ""
+                        }
+                      ],
+                      "filePattern": ".pdf,.doc,.docx,.jpg,.png,.txt",
+                      "fileMaxSize": "4MB",
+                      "multiple": true,
+                      "key": "transportationCostsFile",
+                      "customConditional": "show = (data.transportationCostSituation === 'noLimit' || data.transportationCostSituation === 'special' || data.transportationCostSituation === 'educationPlacement');",
```

Comment:
This condition seems to be obsolete with the removal of an option `I don't require additional transportation allowance`

![image](https://github.com/bcgov/SIMS/assets/54600590/bdd704b3-950b-4189-961c-b7d150284e8b)